examples/foundational/21b-tavus-layer-daily-transport.py

```diff
@@ -87,44 +90,31 @@ async def main():
         task = PipelineTask(
             pipeline,
             params=PipelineParams(
-                # We just use 16000 because that's what Tavus is expecting and
-                # we avoid resampling.
                 audio_in_sample_rate=16000,
-                audio_out_sample_rate=16000,
+                audio_out_sample_rate=48000,
```

Comment:
why 48000? in the new `capture_audio_renderer` the default sample rate is 16000.

Reply:
Before the recent changes in the `capture_participant_audio` we were not able to specify the `sample_rate`, and we were receiving by default 48000 if I am not mistaken. 
So I was testing with it here to see if it would improve the quality.

Reply:
@markbackman changing the `audio_out_sample_rate` to `24000` has highly improved the audio quality. I guess this was probably the sample_rate that Tavus was expecting to receive.